### ISSUE TITLE
Organising the Cluster API (CAPI) Hive Sprint

### ISSUE THREAD

**Issue by teemow** [giantswarm/roadmap#277]
*tl;dr*
We would like to invest 1 month full focus on understanding, coordinating and implementing the company wide move to Cluster API. We have a suggestion on how to approach this. Let’s discuss and fine-tune it together on Wednesday. 

*longer version*
We are currently thinking about a different format to organize and kick off the switch to Cluster API across the KaaS teams plus Biscuit. As you all know the topic is quite important for our KaaS offering this year and there are still many unknowns and risks involved. By making this a more synchronized joint effort that is not necessarily bound to the typical team boundary we might be able to overcome the hurdles more easily.

There are quite a few topics we need to solve in general for each provider team. So it might be more efficient if "Swarms" across teams would solve these issues together for all providers (CAPA, CAPZ, CAPV). 

*The goal is to move to upstream controllers all at once.*

Notes are here: https://docs.google.com/document/d/1guvDx3pKpWXwiW3n2rYx8BIx5IYUk0s7PuiJpp1HHl4/edit#

**Comment by MartaD** [comment]
Anything I can help with?

**Comment by teemow** [comment]
@MartaD yes, thank you! We will brainstorm about that tomorrow in the first session. Saw you added yourself to the event already :+1:

**Comment by annaloew** [comment]
**To Do List (Organizational Part)** 

_Scheduling_
- [ ] Design Day 1 Outline
- [ ] Design Day 2 Outline
- [ ] Design Day 3-5 Outline
- [ ] Design Closing Ceremony Outline, invite all company 
- [ ] Design Moving Stuff back to teams Outline
- [ ] Schedule Speaker Date
- [ ] Create correcsponding Calender Invitations 

 _Scheduling on the fly_
- [ ] Daily Check ins
- [ ] After Sprint Retros 
- [ ] All Hand meetings 
- [ ] Lunch / Fun Activities 
- [ ] Feedback Session on the new format 
- [ ] Create correcsponding Calender Invitations

 _Ad hoc Sessions to offer_
- [ ] How to PO in a Hive 
- [ ] Retros for Hives
- [ ] Wrap Ups for other timezone / holiday/ afk people 
- [ ] WTF is cluster API for Rainmakers and TeamTeam? 

_Communication_
- [ ] asap "how Monday will look like incl. FAQs (Noth-A people, holidays, customer request proxy, support, ops duty) 
- [ ] create slack channel for entire CAPI Sprint and invite 
- [ ] create slack channel for each hive swarm 
- [ ] write Monday 1.3.21 morning reminder and welcome in general 
- [ ] check with team content if interviews of a blog post make sense 
- [ ] define a way, the entire hive gives updates to the rest of the company 

_Miro_
- [ ] Design Materials for the days, esp 1-3 
- [ ] Create spaces for each hive 
- [ ] create a meet-link hub (entire group, per hive swarm, breakout group slide) 
- [ ] Create Agenda with Dates 

_github_
- [ ] Create Github Boards for each hive swarm? 

_well-being_ 
- [ ] organize Lunch 
- [ ] organize a Lunch Walk 
- [ ] draft a how happy are you for this time 
- [ ] do sth nice and secret

**Comment by annaloew** [comment]
***Open Question **Collection***

- North America in one hive swarm? 
- Spread Working group 
- Kernarbeitszeitagreement?

**Comment by MartaD** [comment]
> After Sprint Retros
Wrap Ups for other timezone / holiday/ afk people
define a way, the entire hive gives updates to the rest of the company

Maybe a demo day at the end of a sprint/week? each hive presents progress, accomplishments, issues, and learnings to the rest, sessions recorded for later viewing to afk people so they could join later/not miss a lot?

**Comment by puja108** [comment]
> Create Github Boards for each hive swarm?

I'd say we use one for the whole thing, we could even take over the WG one, but maybe clean slate is better.

**Comment by DrAnnavd** [comment]
I'm not sure that's a great idea. I would want the swarms to be able to
focus on their goals a lot, but let's discuss.

Puja <notifications@github.com> schrieb am Mi., 24. Feb. 2021, 10:46:

**Comment by annaloew** [comment]
**Outline of the CAPI HIVE SPRINT**

**Week 1**
Day 1
- Alignment
- Dividing the focus into individual Hives & forming Swarms
- Think about dependencies and risks & how to mitigate them

Day 2
-Finding the groove within the Swarm
- alignment
- breakdown structure

Day 3-4
Hacking with checkins around lunch & in the evening (times?)

Day 5
- Test your ideas on others
- Retro in swarms & next steps

**Week 2-3**
- Full Hacking with check-ins to share context
- End the week with testing & retro as in week 1
- Speaker Q&A as fits

**Week 4**
- Prepare transition into teams
- Closing ceremony on last day

**Comment by teemow** [comment]
Just for reference here. Here are some things that @MarcelMue wrote down for the last WG meeting: https://docs.google.com/presentation/d/1QZ8Yv_6oLaMDmd6ieBMwykOmh_wwGDzjp9LoVHlHx-g/edit#slide=id.gbd9e97739f_0_3
